sdk/keyvault/keyvault-keys/rollup.config.js

```diff
@@ -44,15 +44,15 @@ function nodeConfig(test = false) {
 function browserConfig(test = false) {
   const baseConfig = {
     input: input,
-    external: ["ms-rest-js", "ms-rest-azure-js"],
+    external: ["@azure/core-http", "@azure/core-lro"],
```

Comment:
For browser builds, neither of these should be external. If users want no duplication, they will need to bundle themselves. Our browser builds come with "batteries included" :)